.github/workflows/regression.yml

```diff
@@ -128,7 +128,7 @@ jobs:
         uses: docker://ghcr.io/su2code/su2/test-su2:221224-1158
         with:
           # -t <Tutorials-branch> -c <Testcases-branch>
-          args: -b ${{github.ref}} -t develop -c develop -s ${{matrix.testscript}}
+          args: -b ${{github.ref}} -t develop -c feature_NEMO_visc_ion -s ${{matrix.testscript}}
```

Comment:
Just a note to not forget this.

Reply:
@WallyMaier Anything we still need to do here?

Reply:
Not until we finally push